tests/functional/codegen/features/decorators/conftest.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def set_initial_balance(env):
+    # set the balance of the deployer so the tests can transfer funds
+    env.set_balance(env.deployer, 10**20)
```

Comment:
let's remove this fixture -- i'd rather have the tests which need funds to explicitly use set_balance